components/config-provider/__tests__/__snapshots__/components.test.js.snap

```diff
@@ -13570,7 +13570,7 @@ Array [
           class="ant-time-picker-panel-input-wrap"
         >
           <input
-            class="ant-time-picker-panel-input  "
+            class="ant-time-picker-panel-input"
```

Comment:
You can drop these changes and rebase master which were fixed already there.

Reply:
Ok, I will do it later.